lib/bot.js

```diff
@@ -430,7 +430,7 @@ module.exports = (function() {
 			});
 		},
 
-		edit: function(title, content, summary, callback) {
+		edit: function(title, content, summary, minor, callback) {
```

Comment:
Let's avoid changing the parameters order. Please support the lack of `minor` argument by checking if it's a function.